oslo/torch/_C/__init__.py

```diff
@@ -64,9 +64,7 @@ def _constant_compatibility_version():
                 universal_newlines=True,
             )
             cuda_version = output.split()
-            cuda_bare_metal_version = cuda_version[
-                cuda_version.index("release") + 1
-            ].split(".")[0]
+            cuda_bare_metal_version = cuda_version[cuda_version.index("release") + 1].split(".")[0]
```

Comment:
How about unifying formatter?
I am using `black`.

Reply:
OK I'll change from autopep8 to black